README.md

```
@@ -59,8 +59,8 @@ Ensure the following artifacts are available in the classpath or loader path:
 - Packet Manager uses configuration files from the **[mosip-config repository](https://github.com/mosip/mosip-config/tree/master)**.
 - Refer to the tagged version corresponding to your release.
 - Key configuration files:
-  - [packet-manager-default.properties](https://github.com/mosip/mosip-config/blob/master/packet-manager-default.properties)
-  - [application-default.properties](https://github.com/mosip/mosip-config/blob/master/application-default.properties)
+    - [packet-manager-default.properties](https://github.com/mosip/mosip-config/blob/master/packet-manager-default.properties)
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)